databases/redis-enterprise-operator/manifests/04-ssl/client-pod-tls.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: redis-client-d
+spec:
+  containers:
+  - name: redis-client-d
+    image: redis:latest
+    command: ["sleep", "3600"]
+    env:
+    - name: PORT 
+      valueFrom:
+        secretKeyRef:
+          name: redb-d-app-db           
```

Comment:
this should be redb-app-db , otherwise the pod will be stuck searching for a non-existent secret

Reply:
Are you sure? We create an additional database with tls enabled for this client: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/926/files/13fa5dc9e33eab83d38e52a87e28e1e3f81817cd#diff-4c2fba2cb70e2f6be6b8e1bdbfc1258499943111092daa8cf536f551b76ca08a

Reply:
And the guide also requests for creating the database:
Enable TLS for incoming connections and internode encryption.
   …
   2	Create the database with enabled TLS:
kubectl apply -n rec-ns -f manifests/04-ssl/d-rdb.yaml